blueocean-dashboard/src/main/js/components/Pipelines.jsx

```diff
@@ -78,6 +68,12 @@ export default class Pipelines extends Component {
                                 })
                             }
                         </Table>
+                        
+                        { pipelines && pipelines.$pager &&
+                            <button disabled={!pipelines.$pager.hasMore} className="btn-show-more btn-secondary" onClick={() => pipelines.$pager.fetchMore()}>
+                                {pipelines.$pending ? 'Loading...' : 'Show More'}
```

Comment:
yeah testing locally its too fast to load, so I guess, do you want to let people just click it a bunch of times to batch it up? or disable while loading... (either... could be argued for if I get the meaning)